language-php.md

```diff
@@ -8,6 +8,69 @@ We decided to follow [PSR-2].
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## SQL Queries
+MySQL keywords are always capitalized: SELECT, INSERT, UPDATE, WHERE, AS, JOIN, ON, IN, etc. Break up long queries into multiple lines for legibility, preferably breaking for each clause.
```

Comment:
I would also concur with @KixPanganiban here. Although my take is to use SQL directly only and only when both these two points occur:

1. There is no dynamic input in the statement.
2. Using the ORM obfuscates the meaning as I find that SQL, when written correctly, is concise and clear.